getid3/module.audio-video.riff.php

```diff
@@ -2030,8 +2032,8 @@ public static function ParseDIVXTAG($DIVXTAG, $raw=false) {
 		if (!$raw) {
 			unset($parsed['genre_id'], $parsed['rating_id']);
 			foreach ($parsed as $key => $value) {
-				if (!$value === '') {
-					unset($parsed['key']);
+				if (empty($value)) {
+					unset($parsed[$key]);
```

Comment:
This bug was introduced in 1.9.4b1
This `if (!$value === '')` comparison using === between bool (`!$value`) and string will always evaluate to false.
On the next line there are `unset($parsed['key'])`, which actually not defined and should be `unset($parsed[$key])`.
Looks like this check was used to filter the empty data, so I've fixed this and this will work correctly from now (this doesn't work at all before this change).